src/main/java/com/couchbase/lite/router/Router.java

```diff
@@ -1507,15 +1521,18 @@ public void changed(Database.ChangeEvent event) {
 
     public Status do_GET_Document_changes(Database _db, String docID, String _attachmentName) {
         // http://wiki.apache.org/couchdb/HTTP_database_API#Changes
-        ChangesOptions options = new ChangesOptions();
+
+        // Get options:
         changesIncludesDocs = getBooleanQuery("include_docs");
-        options.setIncludeDocs(changesIncludesDocs);
         String style = getQuery("style");
-        if (style != null && style.equals("all_docs")) {
-            options.setIncludeConflicts(true);
-        }
-        //options.setContentOptions(getContentOptions());
+        if (style != null && style.equals("all_docs"))
+            changesIncludesConflicts = true;
+
+        ChangesOptions options = new ChangesOptions();
+        options.setIncludeDocs(changesIncludesDocs);
+        options.setIncludeConflicts(changesIncludesConflicts);
         options.setSortBySequence(!options.isIncludeConflicts());
+        // TODO: descending option is not supported by ChangesOptions
```

Comment:
I created ticket for ChangesOptions. https://github.com/couchbase/couchbase-lite-java-core/issues/885